src/DynamicFormControlGroup.vue

```diff
@@ -1,6 +1,9 @@
 <template>
-    <label class="row my-2">
-        <div v-if="controlGroup.label" class="col-form-label col-md-5">
+    <conditional-label
+        :readonly="readonly"
+        :control-group="controlGroup"
+    >
+        <div class="col-form-label col-md-5">
             {{controlGroup.label}}
```

Comment:
It's a little confusing to me for `{{controlGroup.label}}` to be included as part of the slot content to `ConditionalLabel`, since I would have thought it's part of the function of a conditional label to know to include the label on its control group prop if it exists. So I'd be inclined to take it out of the slot content, and have the ConditionalLabel always include it inslide `<label>`, before `<slot>`.

Reply:
As discussed, don't worry about this one - you're right that enclosing div within slot content would mess up the rendering if taking this approach.